include/picongpu/fields/FieldJ.tpp

```diff
@@ -189,8 +189,8 @@ namespace picongpu
     HDINLINE
     FieldJ::UnitValueType FieldJ::getUnit()
     {
-        const float_64 UNIT_CURRENT = UNIT_CHARGE / UNIT_TIME / (UNIT_LENGTH * UNIT_LENGTH);
-        return UnitValueType(UNIT_CURRENT, UNIT_CURRENT, UNIT_CURRENT);
+        const float_64 UNIT_CURRENT_DENSITY = UNIT_CHARGE / UNIT_TIME / (UNIT_LENGTH * UNIT_LENGTH);
```

Comment:
Another reason why we where so confused is that the naming schema is not fitting to our style guide and we thought this is a unit because it is written all caps.
Please change the variable name to `unitCurrentDensity` because this is a simple intermediate variable.

Reply:
@psychocoderHPC fixed